mu/modes/microbit.py

```diff
@@ -344,6 +417,38 @@ def flash_finished(self):
         self.editor.show_status_message(_("Finished flashing."))
         self.flash_thread = None
         self.flash_timer = None
+        if self.python_script:
+            try:
+                self.copy_main()
+            except Exception as ex:
+                self.flash_failed(ex)
+
+    def copy_main(self):
```

Comment:
Out of curiosity more than anything, any reason not to create a temp file and copy it via microfs?

Reply:
IO is slow and tmp files have to be cleaned up? Just a thought

Reply:
Either way seems perfectly fine to me, was asking mostly to get your general view, thanks :)

Reply:
+1